docs/source/execute.rst

```diff
@@ -100,7 +100,7 @@ specify that the ``datalad-pair-run`` orchestrator should be used::
     ./clean.py data/f0.csv cleaned/f0.csv
 
 Notice that in addition to the orchestrator, we specify the input file
-that needs to be available on the remote. If this file were tracked by
+that needs to be available on the remote. If this file was tracked by
```

Comment:
Although not fleshed out, this is describing a case where the file is tracked by git-annex, not Git.  In that case, "were" is correct.  I'll rephrase this to try to avoid confusion.

Reply:
sorry, still missing it -- we are talking about a file (single) not files, so why `were` not `was`?

Reply:
"If I were you, then who would be me?"

Reply:
yeah, I know about that one but I thought it is specifically about "I"("we") and "you" (that one is plural anyways). Google gives 143k for "if file was" vs 8k for "if file were", which doesn't make it correct of cause, but suggests so.

Reply:
The key thing about choosing "was" or "were" with a conditional and a subject that is singular is whether the condition is real or imaginary/hypothetical.

Reply:
I am ok either way if you think that "were" was correct. So far found no clear rule besides "subjunctive mood" but that typically seems to refer to the live subjects, and "if file were" sounded a bit odd to me.

Reply:
I've already reworded it on master: 4739fea9fd